pkg/util/slices/slices_test.go

```diff
@@ -54,3 +54,46 @@ func TestToRefMap(t *testing.T) {
 		})
 	}
 }
+
+func TestToCmpNoOrder(t *testing.T) {
+	cases := map[string]struct {
+		sliceA      []int
+		sliceB      []int
+		expectedRes bool
+	}{
+		"correct_single_occurence_value": {
```

Comment:
Would any of the test cases fail on the previous implementation? IIUC a test case which might be problematic for the previous implementation is: A: {1,2},  B: {1,1}.

Reply:
Yes, the second test "correct_multiple_occurences_value" would fail on the previous implementation, as it didn't distinguish multisets. I'll add your test case as it would also fail on the previous implementation.

Reply:
sgtm, in that case we have 2 test cases that would fail on the previous impl, asserting true and false.